.vsts-ci/windows.yml

```diff
@@ -69,6 +69,26 @@ stages:
       purpose: ElevatedPesterTests
```

Comment:
@SteveL-MSFT, your last commit had 5 failures in `PowerShell-CI-windows`
Enter-PSHostProcess tests.By Process Id.Can enter using NamedPipeConnectionInfo
```powershell
Exception calling "Invoke" with "0" argument(s): "The runspace state is not valid for this operation."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 136
136:                 $ps.AddScript('$pid').Invoke() | Should -Be $pwshId

```
Start-Transcript, Stop-Transcript tests.Transcription should record Write-Information output in correct order when InformationAction is set to Inquire
```powershell
The Win32 internal error "The handle is invalid." 0x6 occurred while retrieving the handle for the active console output buffer. Contact Microsoft Customer Support Services.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.Powershell.Host\Start-Transcript.Tests.ps1: line 230
230:             Write-Information -Message $message -InformationAction Inquire

```
Start-Transcript, Stop-Transcript tests.Transcription should record Write-Host output when InformationAction is set to Continue
```powershell
Transcription cannot be started due to the error: The process cannot access the file 'C:\Users\VssAdministrator\AppData\Local\Temp\5236db99-79a8-47b4-82e4-166d3ad9275e\transcriptdata.txt' because it is being used by another process.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.Powershell.Host\Start-Transcript.Tests.ps1: line 244
244:             Start-Transcript -Path $transcriptFilePath

```
Start-Transcript, Stop-Transcript tests.Transcription should record Write-Host output in correct order when InformationAction is set to Inquire
```powershell
The Win32 internal error "The handle is invalid." 0x6 occurred while retrieving the handle for the active console output buffer. Contact Microsoft Customer Support Services.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.Powershell.Host\Start-Transcript.Tests.ps1: line 290
290:             Write-Host -Message $message -InformationAction Inquire

```
Start-Transcript, Stop-Transcript tests.UseMinimalHeader should reduce length of transcript header
```powershell
Transcription cannot be started due to the error: The process cannot access the file 'C:\Users\VssAdministrator\AppData\Local\Temp\5236db99-79a8-47b4-82e4-166d3ad9275e\transcriptdata.txt' because it is being used by another process.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.Powershell.Host\Start-Transcript.Tests.ps1: line 302
302:             Start-Transcript -Path $transcriptFilePath

```